text/0000-time-improvements.md

```diff
@@ -0,0 +1,332 @@
+- Feature Name: time_improvements
+- Start Date: 2015-09-20
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+This RFC proposes several new types and associated APIs for working with times in Rust.
+The primary new types are `Instance`, for working with time that is guaranteed to be
+monotonic, and `SystemTime`, for working with times across processes on a single system
+(usually internally represented as a number of seconds since an epoch).
```

Comment:
Indeed, in jodatime/threeten, both would be an 'Instant'.

On Sat, Oct 17, 2015, 3:33 PM Cody Schafer <notifications@github.com> wrote:

> In text/0000-time-improvements.md
> <https://github.com/rust-lang/rfcs/pull/1288#discussion_r42314895>:
>
> > @@ -0,0 +1,332 @@
> > +- Feature Name: time_improvements
> > +- Start Date: 2015-09-20
> > +- RFC PR: (leave this empty)
> > +- Rust Issue: (leave this empty)
> > +
> > +# Summary
> > +
> > +This RFC proposes several new types and associated APIs for working with times in Rust.
> > +The primary new types are `Instance`, for working with time that is guaranteed to be
> > +monotonic, and `SystemTime`, for working with times across processes on a single system
> > +(usually internally represented as a number of seconds since an epoch).
>
> Why is it important to mention "across processes on a single system"? How
> does that tell me "this is the one that might move around and tries to be
> as close to absolute time as possible"?
>
> I thought the RFC used Instant as a moment in time, in effect,
> SystemTime::now() would be an Instant, and so would the previous
> ProcessTime::now(). With ProcessTime (aka MonotonicTime) now renamed to
> Instant, what happens to instants (ie: moments in time, as defined
> below)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rust-lang/rfcs/pull/1288/files#r42314895>.
>